apollos-church-api/src/config.js

```diff
@@ -3,8 +3,12 @@ import fetch from 'node-fetch';
 import dotenv from "dotenv/config"; // eslint-disable-line
 import ApollosConfig from '@apollosproject/config';
 
+const configName = process.env.DATABASE_URL
```

Comment:
this won't work. they don't have content in the DB. we need another check in-place

Reply:
said another way, this will select `config.postgres.yml` but those channel IDs are all wrong because their content is not in the DB

Reply:
Should I comment these changes out, since we want it to use config.yml? Or just wait until they are on the shovel, and then this code would work?

Reply:
Actually, I see the PR you put in for templates.